lib/serverengine/daemon_logger.rb

```diff
@@ -58,6 +64,16 @@ def logdev=(logdev)
         old_file_dev.close if old_file_dev
         @logdev = @file_dev
       end
+      if ServerEngine::linux?
+        if logdev.is_a?(String)
```

Comment:
Problematic `@file_dev` is always String class instance?
If not so, we should handle more class instance here.